platform/linux-generic/arch/powerpc/odp_cpu_arch.c

```diff
@@ -1,41 +0,0 @@
-/* Copyright (c) 2015, Linaro Limited
- * All rights reserved.
- *
- * SPDX-License-Identifier:     BSD-3-Clause
- */
-
-#include "config.h"
-
-#include <odp_posix_extensions.h>
-
-#include <stdlib.h>
-#include <time.h>
-
-#include <odp/api/cpu.h>
-#include <odp/api/hints.h>
-#include <odp/api/system_info.h>
-#include <odp_debug_internal.h>
-#include <odp_time_internal.h>
-
-#define GIGA 1000000000
-
-uint64_t odp_cpu_cycles(void)
-{
```

Comment:
Travis is still doing cross-compilation testing for PowerPC, so I don't see why we want to remove these files. There doesn't seem to be any compelling reason for doing so.

Reply:
As the commit log text says: these files are not currently used. These are not used even when building for PowerPC (uses default arch files). As you can see, the patch does not touch any Makefiles or Travis files, just removes these C files and all Travis tests still pass - including the PowerPC build.

Reply:
Ok.